plugin/src/main/java/com/denizenscript/denizen/objects/properties/entity/EntityOpacity.java

```diff
@@ -0,0 +1,53 @@
+package com.denizenscript.denizen.objects.properties.entity;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import org.bukkit.entity.TextDisplay;
+
+public class EntityOpacity extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name opacity
+    // @input ElementTag(Number)
+    // @description
+    // A text display entity's text opacity, from 0 to 255.

```

Comment:
Should include notes about the odd behavior near-zero you observed in the meta.

Reply:
Added a note for that